roles/openshift-networking/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: ocp network operator
+  command: oc get networks.operator.openshift.io cluster -o json
+  register: ocp_network_operator
+
+- name: ocp dns config
+  command: oc get dns cluster -o json
+  register: ocp_dns
+
+- name: network attachment definitions
+  command: oc get network-attachment-definitions -A -o json
+  register: ocp_networkattachments
+
+- name: Set the collected info as facts
+  set_fact:
+    stockpile_openshift_networking:
```

Comment:
I'll recommend splitting them into 3 separate vars,  and add a simple check for if we can collect something and then set accordingly...